src/Sulu/Bundle/MediaBundle/Media/FormatManager/FormatManager.php

```diff
@@ -461,6 +511,7 @@ protected function getImageExtension($fileName)
         switch ($extension) {
             case 'png':
             case 'gif':
+            case 'jpeg':
```

Comment:
Thought so... Well, leave it like this, would be something that can be cleaned up once we have time.